AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -2382,6 +2455,9 @@ function New-ADAccountForStorageAccount {
         [switch]$OverwriteExistingADObject,
 
         [Parameter(Mandatory=$false, Position=7)]
+        [string]$OverwriteADObjectName,

```

Comment:
Not needed. OverwriteExistingADObject implies name overwrite

Reply:
For example: 

Storage Account: storage123
Current Name for the existing AD Object: obj123

If the user performs the Join-Join-AzStorageAccount with the OverwriteExistingADObject Flag, but without specifying the "ADObjectNameOverride" parameter, the name of the existing AD object will always be reset to the storage account, which is "storage123". 

This extra variable helps to check whether the "ADObjectNameOverride" parameter is provided and prevent that situation from happening. I think it's more intuitive for the user to think that not supplying "ADObjectNameOverride" parameter with the flag "OverwriteExistingADObject" = don't want the name of the existing object to be changed instead of letting the AD Object name to be reset to be the storage account name.

Reply:
Fixed. When the old adobject name is different than the new adobject name, prompt a question to check with the user to see if the user wants to replace the adobject name with the new one.